extension/webpack.prod.js

```diff
@@ -0,0 +1,6 @@
+const { merge } = require('webpack-merge');
+const config = require('./webpack.config.js');
+
+module.exports = merge(config, {
+    mode: 'production',
+});
```

Comment:
```suggestion
});

```
nit: add a new line at EOF